docs/attributes-registry/aws-dynamodb.md

```diff
@@ -0,0 +1,29 @@
+# AWS DynamoDB
```

Comment:
For consistency shouldn't we call this file, etc. just by the top-level namespace (i.e. just `aws` in this case) as it is done with all the other namespaces in the registry?

Reply:
Ah right, I wasn't also sure which way we wanted to approach, but I forgot about the DB case. Cool, I will change then :)

Reply:
@AlexanderWert what do you think about the `model/registry` folder? Should we also have a single yaml file there `aws.yaml` or split between all the "things"?

Reply:
Fixed in [5cdeacd](https://github.com/open-telemetry/semantic-conventions/pull/850/commits/5cdeacdfba95dbcfcbdba8f6680697fbecd116fe)